_posts/2016-05-24-25th-anniversary-state-geographic-information-database.md

```diff
@@ -24,12 +24,11 @@ For those of us not in the GIS field at the time, 1991 was a significant year in
 
 In March of 1991, Governor Norman Bangerter signed into law Senate Bill 21, sponsored by Senator Dixie Leavitt. That was 25 years ago this Spring and, nationally, it was one of the, if not the first, instances of GIS being codified into state statute.
 
-[![SGID @ 25]({{ "/images/SGID25YEARS_small.png" | prepend: site.baseurl }})]({{ "/images/SGID25YEARS.png" | prepend: site.baseurl }})
-{: .inline-text-right}
+<div class="caption"><a href ="http://gis.utah.gov/images/SGID25YEARS.png"><img src="{{ "/images/SGID25YEARS_small.png" | prepend: site.baseurl }}" alt="SGID 25th Anniversary Sticker" /><p class="caption-text">Commemorative SGID 25th Anniversary Sticker</p></div>
```

Comment:
remove the `gis.utah.gov` piece. The formatting should be similar to the `img src` tag. you are also missing a closing `</a>` tag.